presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
+* Add `ngrams` function to generate the N-grams from an array.
```

Comment:
Functions are prefixed by `:func:` so that they act as links. Check other release notes.